src/libraries/Directory.Build.props

```diff
@@ -301,7 +258,7 @@
     <!-- Helix properties -->
     <OSPlatformConfig>$(TargetOS).$(Platform).$(Configuration)</OSPlatformConfig>
     <AnyOSPlatformConfig>AnyOS.AnyCPU.$(Configuration)</AnyOSPlatformConfig>
```

Comment:
Lower case AnyOS and AnyCPU as well?

Reply:
AnyCPU comes from VS and this is the casing they use.
`AnyOS` is our invention, and perhaps it should be eliminated.  There is a representation in the RID graph:
https://github.com/dotnet/runtime/blob/1124c1ac95bf1b74b4a030fc21ef0b9f58e33b07/src/libraries/pkg/Microsoft.NETCore.Platforms/runtime.json#L370